bin/run-api.sh

```diff
@@ -2,4 +2,5 @@ cd django-backend
 
 # Run migrations and application
 ./manage.py migrate --no-input --traceback --verbosity 3 &&
-	exec gunicorn --bind 0.0.0.0:8080 fecfiler.wsgi -w 9
+	python manage.py create_committee_views &&
+	exec gunicorn --bind 0.0.0.0:8080 fecfiler.wsgi -w 9
```

Comment:
Should there be a newline at the end of this file?

Reply:
no newline present in the working dev branch